### ISSUE TITLE
Hand drill and jaws of life don't fit into toolbelt

### ISSUE THREAD

**Issue by ontejbjoav** [VOREStation/VOREStation#15432]
#### Brief description of the bug
Hand drill and jaws of life cannot be put into a toolbelt, despite the CE's toolbelt containing them at roundstart

#### What you expected to happen
To be able to put my fancy tools back into the belt after taking them out

#### What actually happened
I couldn't

#### Steps to reproduce
- join as CE
- head to your office and collect your toolbelt
- encounter a situation requiring use of drill/jaws
- attempt to put the tool back and be told that they can't go there

#### Anything else you may wish to add:
![image](https://github.com/VOREStation/VOREStation/assets/69801569/ae75ae61-9f6b-45d5-b080-8e7cdc40ba6d)
![image](https://github.com/VOREStation/VOREStation/assets/69801569/fc77aa22-7b07-4660-9c76-188f53906783)
